docs/data-sources/group.md

```diff
@@ -38,11 +38,15 @@ The following arguments are supported:
 
 The following attributes are exported:
 
+* `allow_external_users` - Indicates whether people external to the organization can send messages to the group. Only set for Unified groups.
```

Comment:
should this be
```suggestion
* `external_users_allowed` - Indicates whether people external to the organization can send messages to the group. Only set for Unified groups.
```
to match the tense of other properties

Reply:
Good catch, will update 👍